.gitignore

```diff
@@ -25,4 +26,14 @@ local-geth/*
 
 # generated files
 scripts/generated
-node_modules/
\ No newline at end of file
+localstate.swagger.json
```

Comment:
Are we sure these need to be ignored?

Reply:
Sorry, this is coming from Kasper's PR branch (events). @kasvtv Do you want me to Are you ok with this

Reply:
Removed legacy gitignore lines on last commit